apps/ledger-live-mobile/src/screens/Onboarding/steps/blePairing.tsx

```diff
@@ -0,0 +1,69 @@
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Flex } from "@ledgerhq/native-ui";
+import BleDevicePairingFlowComponent, {
+  SetHeaderOptionsRequest,
+} from "../../../components/BleDevicePairingFlow";
+import React, { useCallback } from "react";
+import { Device, DeviceModelId } from "@ledgerhq/types-devices";
+import { NavigatorName, ScreenName } from "../../../const";
+import {
+  RootComposite,
+  StackNavigatorProps,
+} from "../../../components/RootNavigator/types/helpers";
+import { OnboardingNavigatorParamList } from "../../../components/RootNavigator/types/OnboardingNavigator";
+import { useSelector } from "react-redux";
+import { hasCompletedOnboardingSelector } from "../../../reducers/settings";
+import { bleDevicePairingFlowHeaderOptions } from "../../BleDevicePairingFlow";
+
+export type Props = RootComposite<
+  StackNavigatorProps<OnboardingNavigatorParamList, ScreenName.OnboardingBlePairing>
+>;
+
+const OnboardingBlePairing: React.FC<Props> = ({ navigation }) => {
```

Comment:
This onboarding is old and the files containing the different components were badly named: it would be better with a capitalized letter 

Can you change this new component and name it `BlePairing.tsx` pls ? Or better :hehe: `BleDevicePairingFlow.tsx` (flow because there is also the scanning that is handled)

Reply:
Yep I wasn't sure about this naming, totally agree np
But it's a flow specified for the Stax, shouldn't we consider this for the naming?